include/glow/IR/IR.h

```diff
@@ -212,6 +228,29 @@ class Instruction : public Value, public TaggedListNode<Instruction> {
 
   static bool classof(const Instruction *I) { return true; }
 
+  /// Dump a textual representation of the Instruction.
+  void dump() const;
+
+  /// Dump to string.
+  std::string toString() const;
+
+  /// Dump a textual representation of the function into provided output stream.
+  void dump(llvm::raw_ostream &OS) const;
+
+  friend llvm::raw_ostream &operator<<(llvm::raw_ostream &os,
+                                       const Instruction &ins) {
+    ins.dump(os);
+    return os;
+  }
+  friend llvm::raw_ostream &operator<<(llvm::raw_ostream &os,
+                                       const Instruction *const &ins) {
```

Comment:
Same here.

Reply:
I think it is better to assert here instead of printing "none"